digitalocean/database/resource_database_cluster_test.go

```diff
@@ -384,6 +384,34 @@ func TestAccDigitalOceanDatabaseCluster_WithVPC(t *testing.T) {
 	})
 }
 
+func TestAccDigitalOceanDatabaseCluster_WithBackupRestore(t *testing.T) {
```

Comment:
This test passes but it gave me some unexpected issues that I still can't make sense of. Commented below.